jekyll/_cci2/getting-started.md

```diff
@@ -15,131 +15,190 @@ This document provides a step-by-step tutorial for getting your first successful
 ## Prerequisites for Running Your First Build
```

Comment:
Could you take out the `2.0` from the intro para above? as there's no other CircleCI?